code/modules/urist/items/uristwar2.dm

```diff
@@ -273,23 +282,24 @@
 	icon_state = "mg42[cover_open ? "open" : "closed"][ammo_magazine ? "" : "-empty"]"
 
 /obj/item/ammo_magazine/a792x57mm/mg42
-	name = "MG 42 belt (7.92x57mm)"
+	name = "MG 42 belt"
 	icon_state = "mg42belt"
 	max_ammo = 250
 	mag_type = MAGAZINE
 
 /obj/item/ammo_magazine/a792x57mm/mg42/empty
 	initial_ammo = 0
 
-/obj/item/grenade/stielhandgranate
+/obj/item/grenade/frag/stielhandgranate
 	desc = "A high explosive grenade, intended to be used as a sort of 'concussion' grenade."
 	name = "Model 21 Stielhandgranate"
 	icon = 'icons/urist/items/uristweapons.dmi'
 	icon_state = "kr_sticknade"
 	item_state = "flashbang"
+	arm_sound = 'sound/effects/flare.ogg' // Closest thing to old fuse sounds.
 
-/obj/item/grenade/stielhandgranate/detonate()
-	explosion(src.loc, 0, 0, 2, 6)
+/obj/item/grenade/frag/stielhandgranate/detonate()
+	explosion(src.loc, 3, EX_ACT_DEVASTATING)
```

Comment:
this isn't supposed to be a frag grenade, and also this seems like too big of an explosion, but also this was probably last used a decade ago, so i don't really care.

Reply:
based, yeah i can change it later, but i doubt we are using it anytime soon